crates/block-producer/src/rpc_client.rs

```diff
@@ -51,7 +51,7 @@ fn to_jsonh256(v: H256) -> JsonH256 {
     h.into()
 }
 
-fn to_result<T: DeserializeOwned>(output: Output) -> anyhow::Result<T> {
+pub fn to_result<T: DeserializeOwned>(output: Output) -> anyhow::Result<T> {
```

Comment:
This is a internal function, should not make it public

Reply:
How about pub(crate) or RPCClient::to_result ?
I place some cell query code only used by withdrawal in withdrawal.rs, it needs to access these code.

Reply:
Done